apps/annotation/opencv_annotation.cpp

```diff
@@ -229,21 +229,29 @@ int main( int argc, const char** argv )
         "{ help h usage ? |      | show this message }"
         "{ images i       |      | (required) path to image folder [example - /data/testimages/] }"
         "{ annotations a  |      | (required) path to annotations txt file [example - /data/annotations.txt] }"
+        "{ maxWindowHeight m  |  -1   | (optional) images larger in height than this value will be scaled down }"
+        "{ resizeFactor r  |  2  | (optional) factor for scaling down [default = half the size] }"
     );
     // Read in the input arguments
     if (parser.has("help")){
         parser.printMessage();
         cerr << "TIP: Use absolute paths to avoid any problems with the software!" << endl;
         return 0;
     }
-    string image_folder(parser.get<string>("images"));
-    string annotations_file(parser.get<string>("annotations"));
+    string image_folder = parser.get<string>("images");
```

Comment:
Basically because I think it is more clear that way, but for efficiency reasons I could indeed change it back!